### PR TITLE
Dependencies for realms-wiki

### PR DESCRIPTION
**Pull request by Mic92**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
I will put all packages into seperate files.

**Comment by Mic92** [comment]
@FRidh I also move out packages out of python-packages.nix here. Can you take a look at some packages and tell me whether you agree on the way I pull in python packages into the scope into external packages.

File: pkgs/development/python-modules/dulwich.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchurl
+, pkgs, gevent, geventhttpclient, mock, fastimport }:
```
Comment (FRidh) [inline pkgs/development/python-modules/dulwich.nix]: not the package set `pkgs` but individual packages

**Review by FRidh — CHANGES_REQUESTED** [review]
Thanks for moving out those packages Just one more change is needed. Aside from that it looks good to me!

File: pkgs/development/python-modules/dulwich.nix (hunk 1 above)
Comment (Mic92) [inline pkgs/development/python-modules/dulwich.nix]: How can I avoid shadowing of packages from `pkgs` by python packages with the same name in this case?

Comment (FRidh) [inline pkgs/development/python-modules/dulwich.nix]: You can explicitly pass the package to `callPackage`, e.g. `inherit (pkgs) git;`,

Comment (Mic92) [inline pkgs/development/python-modules/dulwich.nix]: the reasoning behind passing individual packages is to make it easier to overwrite them?

**Comment by globin** [comment]
@FRidh is specifying python packages in seperate files the prefered syle now?

**Comment by FRidh** [comment]
The file is getting way too big, and the history is also not  useful, so yes.

**Pull request merged**